test/JDBC/expected/BABEL-USER-vu-verify.out

```diff
@@ -203,6 +203,39 @@ babel_user_vu_prepare_test3#!#babel_user_vu_prepare_sch
 ~~END~~
 
 
+ALTER USER babel_user_vu_prepare_test3 WITH LOGIN = babel_user_vu_prepare_test5;
```

Comment:
What's the behaviour when LOGIN name length is > around 60 or 63 chars? Can we add test for that.

Reply:
Again, behavior should not be different from what already exists but I have added one more test case.